__init__.py

```diff
@@ -0,0 +1 @@
+from Brand_Associations.concepts import fibonacci
```

Comment:
[The PEP8 style guide mentions that ](https://peps.python.org/pep-0008/#prescriptive-naming-conventions)
> Modules should have short, all-lowercase names.

Thus, this should be named ```brand_associations```.

Reply:
good tip, thanks!